enterprise/admin/__init__.py

```diff
@@ -101,6 +101,29 @@ def identity_provider(instance):
         ec_idp = instance.enterprise_customer_identity_provider
         return ec_idp and ec_idp.provider_name or ec_idp.provider_id
 
+    def enterprise_catalog(self, instance):
+        """
+        Enterprise catalog id with a link to catalog details page.
+
+        Args:
+            instance (enterprise.models.EnterpriseCustomer): `EnterpriseCustomer` model instance
+
+        Return:
+            catalog id with catalog details link to display in enterprise customer list view.
+        """
+        # Return None if EnterpriseCustomer does not have an associated catalog.
+        if not instance.catalog:
+            return None
+
+        catalog_url = get_catalog_admin_url(instance.catalog)
+        return "{catalog_id}: <a href='{catalog_url}' target='_blank'>View catalog details.</a>".format(
+            catalog_id=instance.catalog,
+            catalog_url=catalog_url,
+        )
+
+    # Allow html tags in enterprise_catalog column, We need to set it true so that anchor tag is not escaped.
```

Comment:
nit: please break it on 80 char length so that eye ball movement is minimal while reading comments